src/components/web3Connection/Account/Account.tsx

```diff
@@ -89,7 +91,12 @@ const Account = (): JSX.Element => {
             <HStack>
               <VStack spacing={0} alignItems="flex-end">
                 <Balance token={communityData.chainData.token} />
-                <Text as="span" fontSize="xs" fontWeight="medium" colorScheme="gray">
+                <Text
+                  as="span"
+                  fontSize="xs"
+                  fontWeight="medium"
+                  color={colorMode === "light" ? "gray.600" : "gray.400"}
```

Comment:
I originally used `colorScheme="gray"` here, since we made that colorScheme a couple of days ago, but I just noticed that you used another color in Figma, so I updated it. Let me know which would work better in your opinion.